<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetBrowser</class>
 <widget class="QDockWidget" name="AssetBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asset browser</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchInput"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="filterComboBox">
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Animation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mesh</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Material</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pipeline</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Physics</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Texture</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Shader</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
